docs/tooling/cli-guides/stop-node.md

```diff
@@ -0,0 +1,38 @@
+# Stop Cloud Server Nodes
+
+This page will guide you to stop a cloud server node set up by Avalanche-CLI
+
+:::warning
+
+ALPHA WARNING: This command is currently in experimental mode. Proceed at your own risk.
+
+:::
+
+## Stopping A Node
+
+To stop all nodes in a cluster, run:
```

Comment:
Let's say the nodes are on a cluster named myValidatorsCluster. To stop all nodes in it, run: